views/peeps.erb

```diff
@@ -0,0 +1,15 @@
+<h1> Welcome to Chipper </h1>
+
+<form action="/newpeep" method="get">
+  <input type="submit" value="Write a new peep" />
+</form>
+
+<form action="/findpeep" method="get">
+  <input type="submit" value="Search for a peep" />
+</form>
+
+<ul>
+  <% @peeps.each do |peep| %>
+    <li> <%= peep.message %> -- posted: <%= peep.date %> </li>
+  <% end %>
+</ul>
```

Comment:
nice, very readable, all user stories met, headings put a smile on my face, what more could I ask for.